### DISCUSSION TITLE
Include Artist in album search

### DISCUSSION THREAD

**Discussion by mcomisso** [tidal-music/discussions#123] [Q&A]
Hi all,
some users were requesting the integration with Tidal, but the API I'm looking for is not really present at the moment.

What I would like to find is the best match from an artist + album search: Spotify and Apple music already support that.

At the moment when I search an album I get a list of results, but there's no way to match a specific artist with them?

Thanks

**Comment by altantidal** [comment]
Hi @mcomisso. At the moment, our API doesn't support searching for albums tied explicitly to a specific artist. However, including the artist's name in your query can help achieve a similar result.

Let us know if you have further questions!

**Reply by mcomisso** [reply]
Thank you for replying! I tried to add the name and album title, but unfortunately the results are all over the place. I'll keep an eye open to see if something improves in the future.
